src/components/textbox/textbox.pw.tsx

```diff
@@ -0,0 +1,1244 @@
+    });
+  });
+
+  test("should render with leftChildren prop", async ({ mount, page }) => {
+    await mount(<TextboxComponentWithLeftChildren />);
+
+    await expect(textbox(page).locator(BUTTON)).toHaveAttribute(
+      "data-component",
+      "button"
+    );
+    await expect(textbox(page).locator(BUTTON)).toBeVisible();
+  });
+
+  testData.forEach((prefix) => {
+    test(`should render with prefix prop as ${prefix}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextboxComponent prefix={prefix} />);
+
+      await expect(textboxPrefix(page)).toHaveText(prefix);
+      await expect(textboxPrefix(page)).toHaveCSS("font-size", "14px");
+      await expect(textboxPrefix(page)).toHaveCSS("font-weight", "900");
+      await expect(textboxPrefix(page)).toHaveCSS("margin-left", "12px");
+    });
+  });
+
+  test("should render with prefix prop whilst align prop is also set to 'right'", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<TextboxComponent prefix="foo" align="right" />);
+
+    await expect(textbox(page)).toHaveCSS("flex-direction", "row");
+  });
+
+  /* This test could be deleted, as its specific test case cannot be properly replicated using Playwright,
+  we can assert it exists in the DOM in the specified DOM structure, but cannot properly test its visibility
+  without added complexity or adding new data tags to `Textbox`s DOM. */
+
+  ([
+    [true, "input", "label"],
+    [false, "label", "input"],
+  ] as [boolean, string, string][]).forEach(
+    ([boolean, firstElement, secondElement]) => {
+      test(`should render with reverse prop set to ${boolean}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TextboxComponent reverse={boolean} />);
+
+        const textboxParentParent = await textbox(page).evaluateHandle(
+          (element: Element) => {
+            return element.parentElement?.parentElement;
+          }
+        );
+
+        const domElement1 = await page.evaluate(
+          ({
+            textboxParentParent: parentDomElement,
+            firstElement: element,
+          }) => {
+            const returnedElement = parentDomElement?.querySelectorAll(element)
+              .length;
+            return returnedElement;
+          },
+          { textboxParentParent, firstElement }
+        );
+
+        const domElement2 = await page.evaluate(
+          ({
+            textboxParentParent: parentDomElement,
+            secondElement: element,
+          }) => {
+            const returnedElement = parentDomElement?.querySelectorAll(element)
+              .length;
+            return returnedElement;
+          },
+          { textboxParentParent, secondElement }
+        );
+
+        expect(domElement1).toBeTruthy();
+        expect(domElement2).toBeTruthy();
+      });
+    }
+  );
+
+  /* This test could be deleted, as its specific test case cannot be properly replicated using Playwright,
```

Comment:
**suggestion**: as mentioned a couple of times above, this comment should be removed, even if the test is decided to be left in